ffms2-sys/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
 #![allow(non_upper_case_globals)]
+#![allow(clippy::upper_case_acronyms)]
```

Comment:
Can you fix `ffms2-sys` clippy lints in another PR, please? Perhaps adding the check to the CI also, thanks!

Reply:
I could do that. But the checks for this pull request would fail, wouldn't they?

Reply:
Right, but first we are going to merge the other PR and then this one, so you can rebase on top of `master` branch

Reply:
Done :)